web/src/settings_config.ts

```diff
@@ -575,6 +575,9 @@ export const preferences_settings_labels = {
     web_escape_navigates_to_home_view: $t({defaultMessage: "Escape key navigates to home view"}),
     web_font_size_px: $t({defaultMessage: "Message-area font size (px)"}),
     web_line_height_percent: $t({defaultMessage: "Message-area line height (%)"}),
+    jump_to_sent_message_conversation: $t({
+        defaultMessage: "Automatically go to conversation where you sent a message",
+    }),
```

Comment:
So two notes. One is that when you add a thing to one of these lists, try to place it in a sorted position.

Second is that we probably want to have a `web_` prefix, since this setting will likely not impact how mobile works. And like every API change, we need to have an "api design" thread about it to confirm the name. I suspect `web_jump_to_sent_message_conversation` is fine, but generally best practice is to start the "api design" discussion as soon as you start working on a feature, so we're not noticing we haven't done one as a PR is otherwise ready to merge.

Reply:
[CZO api design discussion](https://chat.zulip.org/#narrow/stream/378-api-design/topic/Setting.20to.20disable.20.22jump.20to.20sent.20message.20conversation.22/near/1851400)